### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by charles-plessy**
Hello,

this is the first release of the pairgenomealign pipeline!

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pairgenomealign/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/pairgenomealign _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md
Comment (mashehu) [inline README.md:24]:
```suggestion
![pairgenomealign-tubemap.png](docs/images/pairgenomealgn-logo.png)
```
needs to be a markdown image link to be rendered correctly on the website.
Also the tubemap should not replace the logo (which we use for example in the mulitqc report, please restore the old logo there and add this graph as `pairgenomealign-tube-map.png`
(there is also a typo in the current file name)

Comment (charles-plessy) [reply README.md]: I did not see your suggestion and just pushed an equivalent change.

**Review by mashehu — APPROVED** [review]

**Comment by charles-plessy** [comment]
I have a couple of documentation and credit updates that I would like to make, and potential improvements to the MultiQC output.  Shall I wait first for the 1.0.0 release, or can I bring the changes in via PR without invalidating the first approval ?

**Comment by mashehu** [comment]
bring them in and ping me, so I can give you a quick second one 😉

**Comment by charles-plessy** [comment]
Hi @mashehu I have pushed three commits related to MultiQC, acknowledgements and documentation.  On our side we are now ready for 1.0.0 release!  (And I will be absent the next two weeks by the way).

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
You can ask for a review on slack in the #request-review channel

File: CHANGELOG.md
Comment (christopher-mohr) [inline CHANGELOG.md:6]: You might also want to update the date here.

File: subworkflows/local/pairalign_m2m/main.nf
Comment (charles-plessy) [reply subworkflows/local/pairalign_m2m/main.nf:67]: Thanks for the check; I have named the modules according to their output, so yes, `ALIGNMENT_SPLIT_O2M (ALIGNMENT_LASTAL_M2M.out.maf)` is not a typo as it converts a many-to-many alignment to one-to-many.

File: CHANGELOG.md
Comment (charles-plessy) [reply CHANGELOG.md]: Done :)

**Review by christopher-mohr — APPROVED** [review]
LGTM!

**Comment by charles-plessy** [comment]
Thanks!  Can I press the Merge button?

**Comment by mashehu** [comment]
an-older-man-speaking-into-a-microphone-and-saying-go-for-it

**Pull request merged**
